packages/product/src/models/product.ts

```diff
@@ -129,6 +130,7 @@ class Product {
     pivotTable: "product_tags",
     index: "IDX_product_tag_id",
     cascade: ["soft-remove"] as any,
+    // TODO: Do we really want to remove tags if the product is deleted?
```

Comment:
👍🏻 only the pivot entities, not the tag itself.

Reply:
Yeah that's what I would assume as well, but I'll revisit this separately